crates/bevy_window/src/window.rs

```diff
@@ -624,6 +635,17 @@ impl Window {
     }
 }
 
+/// Defines where window should be placed at on creation.
+#[derive(Debug, Clone)]
+pub enum WindowPosition {
+    /// Position will be set by window manager
+    Default,
+    /// Window will be centered at primary monitor
+    Centered,
+    /// Window will be placed at specified position
+    At(Vec2),
```

Comment:
Can we use something like a `LogicalPosition` or other more strongly typed storage here to clarify exactly what's being done?

Reply:
I agree with alice here, it would improve precision.

Reply:
Yeah; I think this is better saved for a follow-up PR. We use raw Vec2 in too many places; I would like to more clearly encode what it means in each case.

Reply:
And I also agree that it should be a separate PR

Reply:
Separate PR makes sense for that.
Marking as resolved then.